pkg/storage/utils/localfs/localfs_unix.go

```diff
@@ -64,3 +64,17 @@ func calcEtag(ctx context.Context, fi os.FileInfo) string {
 	etag := fmt.Sprintf(`"%x"`, h.Sum(nil))
 	return fmt.Sprintf("\"%s\"", strings.Trim(etag, "\""))
 }
+
+func (fs *localfs) GetQuota(ctx context.Context) (uint64, uint64, error) {
+	// TODO quota of which storage space?
+	// we could use the logged in user, but when a user has access to multiple storages this falls short
+	// for now return quota of root
+	stat := syscall.Statfs_t{}
+	err := syscall.Statfs(fs.conf.Root, &stat)
```

Comment:
Can you replace `fs.conf.Root` with `fs.wrap(ctx, "/")`? Sorry, this should have been fixed in the previous PR itself